### ISSUE TITLE
ID of faces detected

### ISSUE THREAD

**Issue by Soxi-T** [chandrikadeb7/Face-Mask-Detection#85]
**PROPOSAL**
A face ID feature that displays the ID to respective faces on the frame. 

Example:

If one person is detected, assign this ID as 0, and display this ID on the bounding box. Whenever a second, third, etc. person enters frame, assign them an ID as 1, 2, and so on, with the ID displaying on the bounding box of their face.

**USE CASE**
This feature could be useful in filtering out false positives, by using alerts by person.

Example:

Person 0 is wearing a mask, person 1 is not. An alert will be created for person 1 (only once, not to spam alerts). Whenever person 0 takes their mask off (with both people still in frame), alert that person 0 is no longer wearing a mask.

Thank you

**Comment by AnshMishra2001** [comment]
Can I work on this issue?

**Comment by Soxi-T** [comment]
> Can I work on this issue?

Of course! I have found a solution using Adrian from pyimagesearch's centroid tracking solution; however, an original solution for this would be great!
